src/backend/database_migrations/versions/20220119_153132_add_sample_accessions_table.py

```diff
@@ -0,0 +1,45 @@
+"""Add sample accessions table
+
+Create Date: 2022-01-10 15:31:32.298296
+
+"""
+import enumtables  # noqa: F401
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "20220119_153132"
+down_revision = "20220114_192608"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+
+    op.create_table(
+        "accessions",
+        sa.Column("sample_id", sa.Integer(), nullable=False),
+        sa.Column("gisaid_isl", sa.String(), nullable=True),
+        sa.PrimaryKeyConstraint("sample_id", name=op.f("pk_accessions_sample_id")),
+        sa.ForeignKeyConstraint(
+            ["sample_id"],
+            ["aspen.samples.id"],
+            name=op.f("fk_accessions_sample_id_samples"),
+        ),
+        schema="aspen",
+    )
+
+    populate_accessions_sql = sa.sql.text(
+        "INSERT INTO aspen.accessions (sample_id, gisaid_isl) SELECT id, NULL FROM aspen.samples"
```

Comment:
This means we'll have an accession row for every sample regardless of whether we have an ISL for it. Is that the intent here? If that's what we want, then we have to make sure we're also automatically creating a new Accession row for every new Sample we add to the db.

I think it's more typical to have a structure for the accessions table like:

```
accessions
--
sample_id
accession_type ENUM('gisaid', 'genbank') # we have a fancy enum library that makes this easy
accession
```

And then only create rows for samples where we find a match in the relevant database. This lets us add support for new databases easily (just update the enum) and the presence/absence of an accessions row lets us know whether we have matching accessions.

Reply:
Yeah, that's a good idea. I'll implement that